packages/atomic/src/components/search/atomic-rating-range-facet/atomic-rating-range-facet.spec.ts

```diff
@@ -0,0 +1,545 @@
+
+describe('atomic-rating-range-facet', () => {
+  let mockedRegisterFacet: Mock;
+  let mockedNumericFacet: NumericFacet;
+  let mockedSearchStatus: SearchStatus;
+  let mockedTabManager: TabManager;
+  let mockedFacetConditionsManager: FacetConditionsManager;
+
+  beforeEach(() => {
+    mockedRegisterFacet = vi.fn();
+    mockedNumericFacet = buildFakeNumericFacet({});
+    mockedSearchStatus = buildFakeSearchStatus({firstSearchExecuted: true});
+    mockedTabManager = buildFakeTabManager({});
+    mockedFacetConditionsManager = buildFakeFacetConditionsManager({});
+  });
+
+  const renderRatingRangeFacet = async ({
+    props = {},
+  }: {
+    props?: Partial<{
+      field: string;
+      facetId: string;
+      label: string;
+      numberOfIntervals: number;
+      maxValueInIndex: number;
+      minValueInIndex: number;
+      icon: string;
+      isCollapsed: boolean;
+      headingLevel: number;
+      filterFacetCount: boolean;
+      injectionDepth: number;
+      tabsIncluded: string[];
+      tabsExcluded: string[];
+      dependsOn: Record<string, string>;
+    }>;
+  } = {}) => {
+    vi.mocked(buildNumericFacet).mockReturnValue(mockedNumericFacet);
+    vi.mocked(buildSearchStatus).mockReturnValue(mockedSearchStatus);
+    vi.mocked(buildTabManager).mockReturnValue(mockedTabManager);
+    vi.mocked(buildFacetConditionsManager).mockReturnValue(
+      mockedFacetConditionsManager
+    );
+
+    const {element} =
+      await renderInAtomicSearchInterface<AtomicRatingRangeFacet>({
+        template: html`<atomic-rating-range-facet
+          field=${props.field ?? 'rating'}
+          label=${props.label ?? 'Rating'}
+          facet-id=${ifDefined(props.facetId)}
+          number-of-intervals=${ifDefined(props.numberOfIntervals)}
+          max-value-in-index=${ifDefined(props.maxValueInIndex)}
+          min-value-in-index=${ifDefined(props.minValueInIndex)}
+          icon=${ifDefined(props.icon)}
+          ?is-collapsed=${props.isCollapsed}
+          heading-level=${ifDefined(props.headingLevel)}
+          ?filter-facet-count=${props.filterFacetCount}
+          injection-depth=${ifDefined(props.injectionDepth)}
+          .tabsIncluded=${props.tabsIncluded || []}
+          .tabsExcluded=${props.tabsExcluded || []}
+          .dependsOn=${props.dependsOn || {}}
+        ></atomic-rating-range-facet>`,
+        selector: 'atomic-rating-range-facet',
+        bindings: (bindings) => ({
+          ...bindings,
+          store: {
+            ...bindings.store,
+            getUniqueIDFromEngine: vi.fn().mockReturnValue('123'),
+            registerFacet: mockedRegisterFacet,
+          },
+        }),
+      });
+
+    const locators = {
+      get title() {
+        return page.getByText(props.label ?? 'Rating', {exact: true});
+      },
+      getFacetValueByPosition(valuePosition: number) {
+        return page.getByRole('listitem').nth(valuePosition);
+      },
+      get clearButton() {
```

Comment:
Nit: this locator is a bit brittle.

I think we could use `page.getByRole('button', {name: /clear.*filter/i})` instead.